docs/howto/mock_services.md

```diff
@@ -78,126 +78,166 @@ ZIO Test provides a framework for mocking your modules.
 
 ## Creating a mock service
 
-Assuming you are following the [module pattern][doc-use-module-pattern], for an `AccountListener` service we put the _capability tags_ and the access
-helpers (within `>` object) in the service companion object:
-
-```scala mdoc:invisible
-trait AccountEvent
-```
+We'll be assuming you've read [How to use modules and layers][doc-use-modules-and-layers] guide. In the main sources we define the _service_, a module alias and _capability accessors_. In test sources we're defining the _mock object_ which will hold _capability tags_ and mock layer machinery.
 
 ```scala mdoc:silent
 import zio.test.mock._
 
-type AccountObserver = Has[AccountObserver.Service]
+// main sources
+type Example = Has[Example.Service]
 
-object AccountObserver {
+object Example {
   trait Service {
-    def processEvent(event: AccountEvent): IO[Nothing, Unit]
+    val static                                 : UIO[String]
+    def zeroArgs                               : UIO[Int]
+    def zeroArgsWithParens()                   : UIO[Long]
+    def singleArg(arg1: Int)                   : UIO[String]
+    def multiArgs(arg1: Int, arg2: Long)       : UIO[String]
+    def multiParamLists(arg1: Int)(arg2: Long) : UIO[String]
+    def command(arg1: Int)                     : UIO[Unit]
+    def overloaded(arg1: Int)                  : UIO[String]
+    def overloaded(arg1: Long)                 : UIO[String]
   }
+}
+
+// test sources
+object ExampleMock {
 
-  object Service {
-    object processEvent extends Method[Service, AccountEvent, Unit]
+  sealed trait Tag[I, A] extends Method[Example, I, A] {
+    val mock: ZLayer[Has[Mock], Nothing, Example] = ???
```

Comment:
Can we do `type Mock = Has[Mock.Service]` like we do with all the other services?